backend/_handler.js

```diff
@@ -4,106 +4,110 @@ import * as db from '@reshuffle/db';
 const swaggerUi = require('swagger-ui-express');
 const swaggerDocument = require('./swagger.json');
 
-
 const app = express();
 
 // API endpoints
 const allPetsQuery = db.Q.filter(db.Q.key.startsWith('pets/'));
 
 //GET /collection - gets all entries in the collection, query string is the filter attributes like /collection?param1=foo
```

Comment:
The documentation says that the query string is used, but it is not passed to the query.